content/docs/0.8.x/operate/upgrade/index.md

```diff
@@ -8,3 +8,359 @@ aliases:
 
 ## Upgrade from Keptn 0.7 to 0.8
 
+1. Before starting the update, we strongly advise to create a backup of your Keptn projects. To do so, please follow the instructions in the [backup guide](../../operate/backup_and_restore).
+
+1. To download and install the Keptn CLI for version 0.8.0, you can choose between:
+    * **Automatic installation of the Keptn CLI (Linux and Mac)**:
+
+      * The next command will download the 0.8.0 release from [GitHub](https://github.com/keptn/keptn/releases), unpack it, and move it to `/usr/local/bin/keptn`.
+```console
+curl -sL https://get.keptn.sh | sudo -E bash
+```
+    
+      * Verify that the installation has worked and that the version is correct by running:
+```console
+keptn version
+```
+
+    * **Manual installation of the Keptn CLI:**
+
+      * Download the release for your platform from the [GitHub](https://github.com/keptn/keptn/releases/tag/0.8.0)
+
+      * Unpack the binary and move it to a directory of your choice (e.g., `/usr/local/bin/`)
+
+      * Verify that the installation has worked and that the version is correct by running:
+```console
+keptn version
+```
+
+1. To upgrade your Keptn installation from 0.7.3 to 0.8.0, the Keptn CLI offers the command: 
+```console
+keptn upgrade
+```
+    
+    * Please [verify that you are connected to the correct Kubernetes cluster](../../troubleshooting/#verify-kubernetes-context-with-keptn-installation)
+before executing this command.
+    
+    * This CLI command executes a Helm upgrade using the Helm chart from: [keptn-installer/keptn-0.8.0.tgz](https://storage.googleapis.com/keptn-installer/keptn-0.8.0.tgz)
+
+**Note:** If you have manually modified your Keptn deployment, e.g., you deleted the Kubernetes Secret `bridge-credentials` for disabling basic auth, the `keptn upgrade` command will not detect the modification. Please re-apply your modification after performing the upgrade.
+
+### Upgrading your projects
+
+In Keptn 0.8.0, a [new format for the shipyard.yaml](https://github.com/keptn/spec/blob/0.2.0/shipyard.md) file has been introduced, and Keptn will not support shipyard files based on the [previous specifiation](https://github.com/keptn/spec/blob/0.1.7/shipyard.md)
+Therefore, if you already have created projects on your previous Keptn installation, you will need to use the `keptn upgrade project --shipyard` command to upgrade each of your existing projects to the new format:
```

Comment:
Can we add a reference to the command docu: /0.8.x/reference/cli/commands/keptn_upgrade/ (at `keptn upgrade project`)